.github/actions/compute-sha256/main.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+)
+
+func main() {
+	if len(os.Args) < 2 {
+		log.Println("Usage: sha256sum <file>")
+		panic("missing argument: path to the file to compute the SHA256 hash")
+	}
+
+	file := os.Args[1]
+	if _, err := os.Stat(file); os.IsNotExist(err) {
+		panic(fmt.Sprintf("file not found: %s", file))
+	}
+
+	data, err := ioutil.ReadFile(file)
+	if err != nil {
+		panic(fmt.Sprintf("failed to read file: %s", file))
+	}
+
+	hash := sha256.Sum256(data)
+	log.Printf("computed sha: %s\n", hex.EncodeToString(hash[:]))
+
+	cmd := exec.Command("/usr/bin/env", "bash", "-c",
```

Comment:
Note: this can be done via `printf`. Given the discussion in https://github.com/slsa-framework/slsa-github-generator/pull/503#issuecomment-1180617335, we may want to use TS instead of Go.